scss/iframes.scss

```diff
@@ -4,3 +4,4 @@
 @use 'utilities';
 @use 'config';
 @use 'iframes/index';
+@use 'initial';
```

Comment:
@jgerigmeyer This was what was causing the wrong fonts to load in the previews. 

They are loading correctly now:

![image](https://user-images.githubusercontent.com/41588129/200648848-db3cc7d7-6117-4ac0-881d-d411363f1b8d.png)

Reply:
If this is being loaded in all example output iframes, it's probably a bit much. I think we have two kinds of iframe:

- the kind where we're generating the visualization, with minimum user styles (like these color swatches) - in which case we need the Herman fonts, and these initial styles make sense to include.
- the kind where authors are generating the output, and we should add as little code as possible. There's no reason to include our fonts or any other initial styles in that output.

Reply:
@mirisuzanne We currently don't load any Herman styles into `@example` iframes, but we have a specific `iframes.scss` file that we load for `@colors`, `@fonts`, `@sizes`, and `@ratios`. (Then we load _all_ Herman styles for `@icons` iframes.)

I was noting that those other annotations (fonts, sizes, ratios) have a limited set of styles (incl Herman fonts etc) already in `iframes/_base.scss`, so I think we maybe want to use that same approach for `@colors` rather than loading in `initial`?

Reply:
@jgerigmeyer Ah, I see now. The markup for the color previews is missing the iframe-specific selectors that are already in `iframes/_base.scss`. No need to load all of 'initial'. I'll clean that up.

Reply:
@jgerigmeyer I edited the color template to use the existing styles in `iframes/_base`. Removed some extra whitespace that was happening for colors that had multiple `color-name`'s too.